content/1_General/Adding_Solution.mdx

```diff
@@ -178,11 +177,12 @@ have to follow all of the recommendations):
 	</Resource>
 </Resources>
 
-<IncompleteSection>
+### General Conventions
 
-additional resources about style?
-
-</IncompleteSection>
+- Try to use descriptive variable names.
+- Do not name a data structure the data structure's name.
+  - For example, don't name `std::set` "set".
```

Comment:
Why this?

Reply:
We voted in our meeting to adhere to these guidelines for the health of the guide.